README.rst

```diff
@@ -537,6 +537,10 @@ Choose from the list of available fixers:
                         with at least one whitespace
                         around.
 
+* **echo_to_print** [contrib]
+                        Converts echo functions to
```

Comment:
`echo` is not a function its a language constructor, same goes for print (IIRC).